lib/rules/input-button-name.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "input-button-name",
```

Comment:
Funky spacing. Did Prettier not run here?

Reply:
How is the spacing funky? It looks normal to me.

https://github.com/dequelabs/axe-core/blob/a2c531dd9bc812a639f742cc9b156d2869748b8a/lib/rules/input-button-name.json#L1-L10

Reply:
Weird. Must have been a Github Refined bug. It looked like indentation was set to 1 space. :man_shrugging:

Reply:
Huh, I don't think it is. I think something funky is going on with Github's formatting. There's different spacing from when I looked at it earlier. 👀